src/Microsoft.AspNetCore.Mvc.TagHelpers/Resources.resx

```diff
@@ -147,4 +147,16 @@
   <data name="PropertyOfTypeCannotBeNull" xml:space="preserve">
     <value>The '{0}' property of '{1}' must not be null.</value>
   </data>
+  <data name="ButtonTagHelper_CannotOverrideFormAction" xml:space="preserve">
```

Comment:
Suggest renaming existing resources e.g. `AnchorTagHelper_CannotOverrideHref` and reusing them here. Resources are `internal` so there's no compat worry. And maintenance will be easier if we keep the numbers and repetitions down.